src/sidebar/components/test/help-panel-test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var angular = require('angular');
+
+describe('helpPanel', function () {
```

Comment:
I've kept this consistent with other components tests for now. I'd be happy to accept a PR which unified the style across all tests in our codebase :)